src/error.js

```diff
@@ -407,13 +407,18 @@ export function getErrorReportData(message, filename, line, col, error,
     return;
   }
 
+  const detachedWindow = !(self && self.window);
```

Comment:
I assume you tested this? Looks right, but I'm not 100% sure. 
Maybe `self.parent` would read better :)

Reply:
I haven't been able to test inside the window that gets detached yet (inside the iframe). On the outside, it's easy to see that once the iframe is removed, the global's `window` property gets removed:

```js
const iframe = document.createElement('iframe');
iframe.srcdoc = '';
document.body.appendChild(iframe);

const c = iframe.contentWindow;
document.body.removeChild(iframe);

console.log(c.window); // => null
```

`parent` would also work.

Reply:
```js
iframe.srcdoc = "<script>setInterval(function(){console.log(self.window)}, 100)</script>"
setTimeout(function() { document.body.removeChild(iframe); }, 1000)
```

may work. But I think your test is fine.